src/services/ComputerMind/ComputerMind.js

```diff
@@ -1,48 +1,61 @@
-
 export class ComputerMind {
-    //ustawianie prawdopodobienstwa
-    setGameLevel(randomObj) {
-        let gameLevelObject = {};
-        let keys = Object.keys(randomObj);
-        let possibleAnsw = randomObj[keys[3]]
-        let correctAnsw =  randomObj[keys[2]]
-        if (document.getElementById('easy').checked) {
-          for (const key of possibleAnsw) {
-            gameLevelObject[key] = 0.25;
-          }  
-        } else if (document.getElementById('medium').checked){ 
-            for (const key of possibleAnsw) {
-              gameLevelObject[key] = 0.20;
-              gameLevelObject[correctAnsw] = 0.40;
-    
-              }
-        } else if (document.getElementById('hard').checked){ 
-            for (const key of possibleAnsw) {
-              gameLevelObject[key] = 0.05;
-              gameLevelObject[correctAnsw] = 0.85;
-    
-              }
-      }   return gameLevelObject    
+  //ustawianie prawdopodobienstwa
+  setGameLevel(nextQuestion) {
+    const gameLevelObject = {};
+    const questionKeys = Object.keys(nextQuestion);
+    const possibleAnswers = nextQuestion[questionKeys[3]]
+    const correctAnswer = nextQuestion[questionKeys[2]]
+    if (document.getElementById('easy').checked) {
+      for (const key of possibleAnswers) {
+        gameLevelObject[key] = 0.25;
+      }
+    } else if (document.getElementById('medium').checked) {
+      for (const key of possibleAnswers) {
+        gameLevelObject[key] = 0.20;
+        gameLevelObject[correctAnswer] = 0.40;
+
+      }
+    } else if (document.getElementById('hard').checked) {
+      for (const key of possibleAnswers) {
+        gameLevelObject[key] = 0.05;
+        gameLevelObject[correctAnswer] = 0.85;
+
+      }
     }
+    return gameLevelObject
+  }
 
 
-    //losowanie odpowiedzi komputera
-    RandomComputerAnswer(randomObject, randomObj) {
-    let i, sum = 0, r = Math.random(), keys = Object.keys(randomObj) 
-     
+  //losowanie odpowiedzi komputera
+  randomComputerAnswer(randomObject, nextQuestion) {
+    let i,
+      sum = 0,
+      randomNumber = Math.random(),
```

Comment:
chodziło mi o to, ze deklaracja każdej zmiennej jest w oddzielnej linii. 
tutaj sum jest zmieniane wiec musi byc zadeklarowane z let, ale pozostałe mogą być const.
można zmienić "i" na nazwę jakąś :)

Reply:
oki, dorzuciłam w oddzielnej linii, 'i' podmienilam na inna nazwe ale dałam z let, bo mi rzuca błędem

Reply:
a faktycznie bo jest używane w pętli.. i tam chyba bym to zadeklarował od razu :) for (let i in randomObject) ...

Reply:
oki, zadeklarowane w pętli